XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -222,6 +236,14 @@ private void EnabledRow(Host host, UpdateType type, int index)
                     return;
                 }
 
+                
+                if (poolOfOne != null && PoolForbidsAutomaticUpdates(poolOfOne))

```

Comment:
@GaborApatiNagy , thanks for this input, then it seems to me we agree that the right order of checks should be: pool partially upgraded; pool unlicensed; automatic updates unsupported; everything else that requires calculation of upgrading sequences. Right?